packages/client/src/runtime/core/engines/client/RemoteExecutor.ts

```
@@ -1,7 +1,9 @@
+import { SpanKind } from '@opentelemetry/api'
 import type { QueryEngineLogLevel } from '@prisma/client-common'
 import { applySqlCommenters, type TransactionOptions } from '@prisma/client-engine-runtime'
 import { PrismaClientKnownRequestError } from '@prisma/client-runtime-utils'
 import { Debug } from '@prisma/debug'
+import type { EngineSpan, EngineTraceEvent, TracingHelper } from '@prisma/internals'
 import type { EngineTraceEvent, TracingHelper } from '@prisma/instrumentation-contract'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'EngineTraceEvent'. Consider to delete it or rename it.

'EngineTraceEvent' is defined here:


(lint/suspicious/noRedeclare)